docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+
+volumes:
+    data:
+        driver: local
+    cache:
+        driver: local
+    logs:
+        driver: local
+
+services:
+    sylius:
+        build:
+            context: .
+            dockerfile: ./etc/docker/php/Dockerfile
+        entrypoint: /var/www/etc/docker/php/entrypoint.sh
+        volumes:
+            - .:/var/www
+            - data:/var/www/sylius
+            - cache:/var/www/app/cache
+            - logs:/var/www/app/logs
+        depends_on:
+            - database
```

Comment:
Can we still access the database from this container while using `depends_on` instead of `links`?